Limit displayed chat suggestions to the first two

Nathaniyell opened the pull request:
Limit displayed chat suggestions to the first two for improved user experience in Suggestions component

Nathaniyell committed:
Limit displayed chat suggestions to the first two for improved user experience in Suggestions component

sweetmantech commented on components/Chat/Suggestions.tsx:
```diff
@@ -3,10 +3,11 @@ import SuggestionButton from "../LandingPage/SuggestionButton";
 
 const Suggestions = () => {
   const { suggestions } = useChatProvider();
+  const limitedChatSuggestions = suggestions.slice(0, 2);
```
Go further upstream.
- where are these being generated?
- can you modify the generation to only return `2 results` instead of `splicing` once we've already requested `4 results`?

sweetmantech commented on components/Chat/Suggestions.tsx:
```diff
@@ -3,10 +3,11 @@ import SuggestionButton from "../LandingPage/SuggestionButton";
   return (
     <div className="flex flex-col lg:flex-row items-start space-y-4 lg:space-x-4 mb-2 lg:items-center lg:space-y-0">
-      {suggestions.map((suggestion) => (
+      {limitedChatSuggestions.map((suggestion) => (
```
Personally, I do not think you need to modify this file to properly implement this solution.
- look deeper into `useChatProvider` to see how prompts are generated.

Nathaniyell committed:
Refactor question limit in prompts and update suggestions display

- Reduced the maximum number of questions from 4 to 2 in the prompts API to enhance clarity and focus.
- Reverted the suggestion display in the Suggestions component to show all available suggestions instead of limiting to the first two, improving user access to options.
